### PR TITLE
Works (though unfinished)

### PR DESCRIPTION
**Pull request by AmitaiB**
##tl;dr
This code works, but be aware that it touches the info.plist, podfile, and Appdelegate.m
It now successfully launches a google map, and loads the user's current location.

###Specifically, this commit...
-...adds a cocoapod or two.
-...adds `[GMSServices provideAPIKey...]`
-...updates the tourViewFlow (which shouldn't be a problem, of course)...

###Issues/Next Steps
-The location takes 5-15 seconds to retrieve, and the map hangs while it waits. I will move the location call to the TabBarViewController, but it might even be a good idea to get the location (asynchronously, of course) when loading the app.
-Testing out the UIControl YALFoldingTabBarController...

### CODE REVIEW

**Comment by AmitaiB** [comment]
I take it back: I'll pull in an hour or so, when it works the way I intend.
